broker/src/main/java/io/camunda/zeebe/broker/logstreams/AtomixAppendListenerAdapter.java

```diff
@@ -25,6 +35,17 @@ public void onWrite(final IndexedRaftLogEntry indexed) {
 
   @Override
   public void onWriteError(final Throwable error) {
+    if (error instanceof NoSuchElementException || error instanceof NoLeader) {
+      // Not a failure. It is probably during transition to follower.
+      Broker.LOG.debug(
```

Comment:
:x: Either use its own logger or one from `Loggers`. I think it's just confusing if it uses the `Broker` class' logger.

Reply:
Yeah was also not sure about that. I guess we still have no real policy here.